oracle-plugin/src/main/java/io/cdap/plugin/oracle/OracleConnectorConfig.java

```diff
@@ -103,6 +109,10 @@ public String getDatabase() {
     return database;
   }
 
+  public Boolean getSSlMode() {
+    return useSSL;
```

Comment:
For existing pipelines this property will be null, please handle that case.

Reply:
Done, please have a look if this change would be enough to avoid failures in case of existing pipelines. 

[https://github.com/data-integrations/database-plugins/pull/515/commits/b68ec933ff27ff2a1ade2c578727113604803367#diff-afe65e631b8b5b1d360af43ae84dc844e2c55ee83e9179cdb8bd087da954ae06R114]